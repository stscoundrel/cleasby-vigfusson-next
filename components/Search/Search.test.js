import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Seach from './index'

describe('Search component', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Seach />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<Seach />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
