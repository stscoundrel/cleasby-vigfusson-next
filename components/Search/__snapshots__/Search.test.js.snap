// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component Matches snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    name="search"
    onChange={[Function]}
    type="text"
    value=""
  />
  <button
    type="submit"
  >
    Search
  </button>
</form>
`;
